src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -108,12 +108,16 @@ def _reset_tracking(self):
   def apply_extension_fields(self, extension_fields):
     """Applies _ext_ prefixed extension fields."""
     if extension_fields.get('_ext_collaborators'):
+      self._changed.add('_ext_collaborators')
       for collaborator in extension_fields['_ext_collaborators']:
         self._collaborators.add(collaborator)
-    self._issue_access_limit = extension_fields.get(
-        '_ext_issue_access_limit') or ({
-            'access_level': IssueAccessLevel.LIMIT_NONE
-        })
+
+    if extension_fields.get('_ext_issue_access_limit'):
+      self._changed.add('_issue_access_limit')
+      self._issue_access_limit = extension_fields.get(
```

Comment:
Do not need the `get(..)` and the `or` here.

This can directly be `self._issue_access_limit = extension_fields['_ext_issue_access_limit']`

Might need an else for `self._issue_access_limit = {'access_level': IssueAccessLevel.LIMIT_NONE}`

Reply:
Ah, yes, thanks!

We shouldn't need an 'else' statement for the default, as we set that default on the issue at initialization.